lmnet/tests/lmnet_tests/datasets_tests/test_mscoco.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import pytest
+
+from lmnet.datasets.mscoco import MscocoObjectDetection, DEFAULT_CLASSES
+
+
+pytestmark = pytest.mark.usefixtures("set_test_environment")
+
+
+def test_mscoco_object_detection_optional_classes_instantiate():
+    classes = [DEFAULT_CLASSES[0]]
+    MscocoObjectDetection.classes = classes
+    try:
+        MscocoObjectDetection()
+        assert True
```

Comment:
@takecore 
Thank you for your contribution.
Is it possible to add test for `MscocoObjectDetection` instance?

Reply:
@iizukak 
This test only check for instantiation because this is the PR for #523. 

If there are any problems, I would like to solve it in another PR!! 🦀

Reply:
I see